src/main/webapp/style.css

```diff
@@ -1,10 +1,174 @@
+/* Texts */
+h3 {
+  color:orange;
+  font-family: 'Massenet', helvetica, sans-serif;
+  font-size: 40px;
+  letter-spacing: 3px;
+  text-decoration: underline;
+  text-transform: uppercase;
+}
+
+.about-text {
+  color: black;
+  font-size: 24px;
+}
+
+/* Frame */
+.frame {
+  background-color: white;
+  position: fixed;
+  z-index: 990;
+}
+
+#frame-top {
+  height: 15px;
+  left: 0;
+  top: 0;
+  width: 100%;
+}
+
+#frame-right {
+  bottom: 0;
+  right: 0;
+  top: 0;
+  width: 15px;
+}
+
+#frame-bottom {
+  bottom: 0;
+  left: 0;
+  height: 15px;
+  width: 100%;
+}
+
+#frame-left {
+  bottom: 0;
+  left: 0;
+  top: 0;
+  width: 15px;
+}
+
+/* Navigation Bar */
+.navbar {
+  background: white;
+  height: 138px;
+  left: 0;
+  margin-bottom: 20px;
+  position: fixed;
+  right: 0;
+  top: 0;
+}
+
+.navbar .container {
+  padding-left: 75px;
+  padding-right: 15px;
+  padding-top: 50px;
+  position: relative;
+}
+
+.navbar-title {
+  float: left;
+  font-family: 'Massenet', helvetica, sans-serif;
+  font-size: 50px;
+  color: black;
+  position: static;
+  text-decoration: none;
+  top: auto;
+  transform: none;
+  width: 220px;
+}
+
+.navbar-menu-container {
+  box-sizing: border-box;
+  display: block;
+  overflow: visible;
+}
+
+.navbar-menu {
+  display: block;
+  float: right;
+  font-family: 'Massenet', helvetica, sans-serif;
+  font-size: 25px;
+  margin-right: 30px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.fa {
```

Comment:
Are you using .fa for anything? I can't find it in the html. If you do intend to use it, consider changing the name to something less ambiguous.